3-lending/.gitignore

```diff
@@ -1,13 +0,0 @@
-# Visual studio code
```

Comment:
@unghuuduc FYI: you should not delete the `.gitignore` file in the base lending directory. No worries, I will add it again.

Reply:
Oh right, I forgot it, thanks!